firestore-translate-text/functions/src/logs/messages.ts

```diff
@@ -1,8 +1,13 @@
 export const messages = {
+  backfillComplete: (successCount: number, errorCount: number) =>
+    `Finshed backfilling translations. ${successCount} documents checked, ${errorCount} errors.`,
```

Comment:
Is this accurate? `$successCount documents checked` shouldn't it be "$successCount documents translated, ${errorCount} errors" Or does successCount count input -> target language for each target language? Maybe we should have an "already translated" too once we skip existing translations.

Reply:
Touched up the wording here. Now, this is showing the total number of docs translated sucessfully (into all languages) and the total number of docs that had some sort of error (ie successfully translated into 3 of 4 languages).